data/strings/strings.txt

```diff
@@ -8676,7 +8683,7 @@
     fa = مسیر به
     fi = Reitin loppupiste
     fr = Itinéraire vers
-    hu = Ide
+    hu = Célpont
```

Comment:
az innen ide szerintem érthetőbb és közvetlenebb

Reply:
Szerintem pont fordítva, aki most ismerkedik meg az Organic Maps-el az nem feltétlen tudja, az a gomb mit takar. Én is így jártam. Szerintem a célpont és kiindulópont használata bár nem közvetlenebb, de sokkal egyértelműbb mint a jelenlegi.